### DISCUSSION TITLE
Does this browser work on macOS?

### DISCUSSION THREAD

**Discussion by Macko-pp** [yessGlory17/programmer-browser#22] [Q&A] [answered]
Hi! I'm a beginner developer working on mac and mainly using Firefox DE as my main browser. I would largely benefit from this minimal popup browser as it takes up virtually no space on-screen. I went to download the browser and it downloaded a .deb file, I tried to open it, unpackaged it, or install it. I was finally able to open it but I saw no executable files. So I gave up and assumed it was only possible on Linux. Is there any way to download it for mac? If not, do you know any similar alternatives?

Thanks in advance!

**Comment by yessGlory17** [comment, marked as answer]
Currently it only supports windows and linux. I don't know of an alternative. But I plan to support macos next month.
